wrappers/HumanDynamicsWrapper/HumanDynamicsWrapper.cpp

```diff
@@ -8,7 +8,7 @@
 
 #include "HumanDynamicsWrapper.h"
 #include <hde/interfaces/IHumanDynamics.h>
-#include <HumanDynamicsEstimation/HumanDynamics.h>
+#include <hde/msgs/HumanDynamics.h>>
```

Comment:
```suggestion
#include <hde/msgs/HumanDynamics.h>
```

Reply:
@lrapetti  but with this typo, how it was building successfully!?